torchvision/transforms/functional.py

```diff
@@ -577,3 +577,38 @@ def to_grayscale(img, num_output_channels=1):
         raise ValueError('num_output_channels should be either 1 or 3')
 
     return img
+
+
+def erase(img, x, y, w, h, value=0):
+    """ Erase the given Image with random values,
+        or the Imagenet mean pixel value.
+        'Random Erasing Data Augmentation' by Zhong et al.
+        See https://arxiv.org/pdf/1708.04896.pdf
+
+    Args:
+        img (Tensor): Image to be erased.
+        x: Upper pixel coordinate.
+        y: Left pixel coordinate.
+        w: Width of the erased image.
+        h: Height of the erased image.
+        values: Type of erasing value.
+                If values = 0, erasing with random values,
+                else, erasing with the Imagenet mean pixel value.
+
+    Returns:
+        Erased Image.
+    """
+    imagenet_mean_value = [0.485, 0.456, 0.406]
+    if img.size()[0] == 3:
```

Comment:
Instead of having this condition, I think it would be better to just take the size of the tensor directly. Something like
```python
img[:, y:y + h, x:x + w] = torch.rand(img.size(0), h, w)
```
?

But, thinking about it again, there is *randomness* in this function (because of `torch.rand`), and I think it might be better to let the user pass the value that he wants to use as masking as an argument. So it became a single line
```python
img[:, y:y + h, x:x + w] = value
```
And if value is a `(img.size(0), 1, 1)` tensor, then it gets broadcasted naturally to fit your needs